lexicons/app/bsky/feed/postgate.json

```diff
@@ -16,16 +16,16 @@
             "format": "at-uri",
             "description": "Reference (AT-URI) to the post record."
           },
-          "detachedQuotes": {
+          "detachedEmbeds": {
             "type": "array",
             "maxLength": 50,
             "items": {
               "type": "string",
               "format": "at-uri"
             },
-            "description": "List of detached quote post URIs."
+            "description": "List of detached embed URIs."
```

Comment:
'embedding' right? These are the URIs of posts that embed the post modified by this postgate

Reply:
+1